packages/components/src/components/optgroup/optgroup.styles.ts

```diff
@@ -52,4 +52,8 @@ export default css`
     color: var(--syn-color-neutral-800);
     font-size: var(--syn-spacing-large);
   }
+
+  .optgroup__options ::slotted(syn-option[hidden]) {
```

Comment:
Just curious, where is this coming from? Should an attribute of hidden noch automatically hide the item? If this is for hiding empty optgroups I think it should instead be a `has` selector, but those don´t work with slotted unfortunately :(

Reply:
We have the components.style.ts which handles that elements with the attribute `hidden` on them are not displayed. But unfortunately this is not done for slotted elements just with elements in the shadowRoot:
<img width="634" alt="Bildschirmfoto 2024-12-20 um 14 14 53" src="https://github.com/user-attachments/assets/22fb55bf-4b59-4bd9-86e6-03cca4e197f2" />

If not doing this, options are visible although they should not be, if there is another option in an optgroup which IS visible. See following gif: 
![optgroup](https://github.com/user-attachments/assets/13c2556f-ec02-4e9d-b51b-157eb64ef39c)